app/services/payment_requests/create_service.rb

```diff
@@ -10,19 +10,34 @@ def initialize(organization:, params:)
     end
 
     def call
-      return result.not_found_failure!(resource: "customer") unless customer
+      # TODO: Return error unless premium license
 
-      ActiveRecord::Base.transaction do
-        # TODO: Create payable group for the overdue invoices
+      return result.not_found_failure!(resource: "customer") unless customer
+      return result.not_found_failure!(resource: "invoice") if invoices.empty?
 
-        # TODO: Create payment request for the payable group
+      # TODO: Return error if invoices are not overdue
 
-        # TODO: Send payment_request.created webhook
+      ActiveRecord::Base.transaction do
+        # NOTE: Create payable group for the overdue invoices
+        payable_group = customer.payable_groups.create!(organization:)
+        invoices.each { |i| i.update!(payable_group:) }
+
+        # NOTE: Create payment request for the payable group
+        payment_request = payable_group.payment_requests.create!(
+          organization:,
+          customer:,
+          amount_cents: invoices.sum(:total_amount_cents),
+          amount_currency: invoices.first.currency,
+          email:
+        )
+
+        # NOTE: Send payment_request.created webhook
+        SendWebhookJob.perform_later("payment_request.created", payment_request)
```

Comment:
we are not testing the `SendWebhookJob` is queued.

Reply:
Good spot 👍